apps/src/lib/script-editor/LevelToken.jsx

```diff
@@ -130,7 +131,7 @@ class LevelToken extends Component {
             style={Object.assign({}, styles.levelToken, {
               transform: `translate3d(0, ${y}px, 0) scale(${scale})`,
               boxShadow: `${color.shadow} 0 ${shadow}px ${shadow * 3}px`,
-              zIndex: this.props.drag ? 1000 : 500 - this.props.level.position
+              zIndex: draggedLevelPos ? 1000 : 500 - this.props.level.position
```

Comment:
Why 1000 and 500?

Reply:
just to be in front of everything else on the page. FWIW these numbers are unchanged in this PR, but always taking a second look at lines when touching them